stacks/dbrs/cassandra/Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM clearlinux/stacks-clearlinux:latest
+MAINTAINER otc-swstacks@intel.com
+
+RUN swupd bundle-add curl java-runtime python2-basic which pmdk sudo
```

Comment:
Ah this reminds me of a general problem that needs to be scrubbed but if this requires python2 then I would -1. python2 is going to be removed from the distro completely in a few months so I don't want any more new users of it.

Reply:
There is a reported unresolved issue related to the Cassandra client depending on python 2: https://issues.apache.org/jira/browse/CASSANDRA-10190.

Reply:
Ah that's currently not working today in Clear then. Would you be able to submit the work in progress patches to our cassandra package for now? It seemed like from that thread the patches are working just some small nit picks and test case issues.

Reply:
We are not using Cassandra from ClearLinux, it's a Cassandra version from pmem team. We are also planning to get rid of python2 in a next release.

Reply:
Ah okay. When is the next release?

Reply:
so our first release is planned to happen next monday (hopefully) and we can plan our next release to happen anytime, probably after getting rid of this pyhon2 dependency and incorporating the new bundle you created for numa (and get rid of os-testsuite-phoronix-server)

Reply:
Okay, seems like a decent path to not breaking.